### PR TITLE
asadiqbal08/Zendesk Theme updates

### PR DESCRIPTION
**Pull request by asadiqbal08**
#### Acceptance Criteria:

- [x] changes made in fork of Zendesk theme repo at https://github.com/mitodl/mitxpro-zendesk-theme
- [x] Set font to match MITx Pro site
- [x] "Contact Us" button links to new request form, https://xpro.zendesk.com/hc/en-us/requests/new
- [x] Styled as per design.
- [x] Added the Rajdhani font in the selector and mark it default for that theme.
Screen Shot 2019-06-11 at 4 36 23 PM

- [x] Re order the `Contact US` link in the header.
- [x] Updated the default translations file `translations/en-us.json` for labels and descriptions text.
- [x] Utilize the `manifest.json` file and provide content override support via admin panel.
- [x] Footer section contents are totally re configurable via Zendesk theme admin.
   - Contact us heading can be changed/modified via admin.
   - Contact us can be hide/show via admin.
   - MIT Address & Phone number can be changed via admin. 
   - MIT resource pages URL's are configurable via admin. 

#### 1- Page Design:
 screencapture-xpro-zendesk-hc-en-us-2019-06-11-16_08_14

#### 2-  Theme Panel:
Screenshot_2019-06-11 Customize design

@pdpinch FYI over it.

### CODE REVIEW

**Comment by pdpinch** [comment]
I know this is still WIP, but I wanted to point out that the screenshot from 6/10 includes a "community" section and a "recent activity" section that we don't plan to offer. ("Recent activity" may be limited to ZenDesk agents, in which case it's not worth putting in effort to hide).

**Comment by asadiqbal08** [comment]
@pdpinch 
- For community section, One can disable it via Help Center, that is the corresponding condition in template. 
```
{{#if help_center.community_enabled}}
```
- For `Recent Activities`, I have disabled it in `manifest.json` file be default for that theme. However, A person can enable it again via visiting admin screen below for that theme. 

Screen Shot 2019-06-11 at 1 51 41 PM

**Comment by asadiqbal08** [comment]
@pdpinch I have updated the acceptance criteria + screenshots for #273 above. 
Added details that are the part of this PR.

File: templates/home_page.hbs [hunk 1]
```diff
@@ -11,9 +12,20 @@
...
+              <div class='category-image'>
+                {{#is name 'COURSES & PROGRAMS'}}
+                    <img src="{{asset 'icon-program.png'}}" />
+                {{/is}}
+                {{#is name 'PAYMENTS & REFUNDS'}}
+                    <img src="{{asset 'icon-payment.png'}}" />
+                {{/is}}
+                {{#is name 'GENERAL FAQs'}}
+                    <img src="{{asset 'icon-faqs.png'}}" />
+                {{/is}}
```
Comment (asadiqbal08) [inline templates/home_page.hbs:24]: I am thinking for a default image if nothing executed over there.

File: styles/_footer.scss [hunk 2]
```diff
@@ -1,20 +1,167 @@
...
+    &:before {
+      position: absolute;
+      left: 0;
+      bottom: 0;
+      right: 0;
+      content: '';
+      height: 30px;
+      background: rgba(black, .3);
+    }
+    h1 {
```
Comment (ahmed-arbisoft) [inline styles/_footer.scss:159]: Newline before block.

File: styles/_footer.scss [hunk 3]
```diff
@@ -1,20 +1,167 @@
...
+    &:after {
+      position: absolute;
+      left: 0;
+      top: 0;
+      right: 0;
+      content: '';
+      height: 10px;
+      background: linear-gradient(to right, #3dbdbf 0%, #f1ba46 100%);
+    }
+    &:before {
```
Comment (ahmed-arbisoft) [inline styles/_footer.scss:149]: Newline before block.

File: styles/_footer.scss [hunk 4]
```diff
@@ -1,20 +1,167 @@
...
+    font-size: 14px;
+    line-height: 20px;
+  }
+  .contact-block {
+    text-align: center;
+    color: black;
+    padding: 30px 0 80px;
+    position: relative;
+    background: #fff url($mit_background) repeat-x 50% 100%;
+    &:after {
```
Comment (ahmed-arbisoft) [inline styles/_footer.scss:139]: Newline before block.

File: styles/_footer.scss [hunk 5]
```diff
@@ -1,20 +1,167 @@
...
+      }
+    }
+  }
+  .copyright {
+    display: block;
+    text-align: center;
+    font-size: 14px;
+    line-height: 20px;
+  }
+  .contact-block {
```
Comment (ahmed-arbisoft) [inline styles/_footer.scss:132]: Newline before block.

File: styles/_footer.scss [hunk 6]
```diff
@@ -1,20 +1,167 @@
...
+        content: "|";
+      }
+      &:first-child {
+        &::before {
+          display: none;
+        }
+      }
+    }
+  }
+  .copyright {
```
Comment (ahmed-arbisoft) [inline styles/_footer.scss:125]: Newline before block.

File: styles/_footer.scss [hunk 7]
```diff
@@ -1,20 +1,167 @@
...
+      display: inline-block;
+      vertical-align: top;
+      font-size: 15px;
+      line-height: 20px;
+      &::before {
+        padding: 0 5px;
+        content: "|";
+      }
+      &:first-child {
+        &::before {
```
Comment (ahmed-arbisoft) [inline styles/_footer.scss:118]: Newline before block.

File: styles/_footer.scss [hunk 8]
```diff
@@ -1,20 +1,167 @@
...
+    li {
+      display: inline-block;
+      vertical-align: top;
+      font-size: 15px;
+      line-height: 20px;
+      &::before {
+        padding: 0 5px;
+        content: "|";
+      }
+      &:first-child {
```
Comment (ahmed-arbisoft) [inline styles/_footer.scss:117]: Newline before block.

File: styles/_footer.scss [hunk 9]
```diff
@@ -1,20 +1,167 @@
...
+    margin: 0 0 10px;
+    list-style: none;
+    text-align: center;
+    padding: 10px 0 0;
+    li {
+      display: inline-block;
+      vertical-align: top;
+      font-size: 15px;
+      line-height: 20px;
+      &::before {
```
Comment (ahmed-arbisoft) [inline styles/_footer.scss:113]: Newline before block.

File: styles/_footer.scss [hunk 10]
```diff
@@ -1,20 +1,167 @@
...
+      width: auto;
+    }
+  }
+  .footer-sub-nav {
+    padding: 0;
+    margin: 0 0 10px;
+    list-style: none;
+    text-align: center;
+    padding: 10px 0 0;
+    li {
```
Comment (ahmed-arbisoft) [inline styles/_footer.scss:108]: Newline before block.

File: styles/_footer.scss [hunk 11]
```diff
@@ -1,20 +1,167 @@
...
+    @include tablet {
+      float: left;
+      width: 100%;
+    }
+    @include desktop {
+      float: none;
+      width: auto;
+    }
+  }
+  .footer-sub-nav {
```
Comment (ahmed-arbisoft) [inline styles/_footer.scss:101]: Newline before block.

File: styles/_footer.scss [hunk 12]
```diff
@@ -1,20 +1,167 @@
...
+    }
+  }
+  .center-content {
+    text-align: center;
+    overflow: hidden;
+    @include tablet {
+      float: left;
+      width: 100%;
+    }
+    @include desktop {
```
Comment (ahmed-arbisoft) [inline styles/_footer.scss:95]: Newline before block.

File: styles/_footer.scss [hunk 13]
```diff
@@ -1,20 +1,167 @@
...
+      }
+      a {
+        color: white;
+      }
+    }
+  }
+  .center-content {
+    text-align: center;
+    overflow: hidden;
+    @include tablet {
```
Comment (ahmed-arbisoft) [inline styles/_footer.scss:90]: Newline before block.

File: styles/_footer.scss [hunk 14]
```diff
@@ -1,20 +1,167 @@
...
+      margin: 0 15px;
+      @include tablet {
+        margin: 0 0 0 30px;
+      }
+      a {
+        color: white;
+      }
+    }
+  }
+  .center-content {
```
Comment (ahmed-arbisoft) [inline styles/_footer.scss:86]: Newline before block.

File: styles/_footer.scss [hunk 15]
```diff
@@ -1,20 +1,167 @@
...
+    li {
+      display: inline-block;
+      vertical-align: top;
+      font-size: 20px;
+      line-height: 28px;
+      margin: 0 15px;
+      @include tablet {
+        margin: 0 0 0 30px;
+      }
+      a {
```
Comment (ahmed-arbisoft) [inline styles/_footer.scss:80]: Newline before block.

File: styles/_footer.scss [hunk 16]
```diff
@@ -1,20 +1,167 @@
...
+    @include tablet {
+      float: right;
+    }
+    li {
+      display: inline-block;
+      vertical-align: top;
+      font-size: 20px;
+      line-height: 28px;
+      margin: 0 15px;
+      @include tablet {
```
Comment (ahmed-arbisoft) [inline styles/_footer.scss:76]: Newline before block.

File: styles/_footer.scss [hunk 17]
```diff
@@ -1,20 +1,167 @@
...
+  }
+  .footer-nav {
+    margin: 0;
+    padding: 0;
+    list-style: none;
+    text-align: center;
+    @include tablet {
+      float: right;
+    }
+    li {
```
Comment (ahmed-arbisoft) [inline styles/_footer.scss:69]: Newline before block.

File: styles/_footer.scss [hunk 18]
```diff
@@ -1,20 +1,167 @@
...
+      margin: 0;
+      font-style: normal;
+    }
+  }
+  .footer-nav {
+    margin: 0;
+    padding: 0;
+    list-style: none;
+    text-align: center;
+    @include tablet {
```
Comment (ahmed-arbisoft) [inline styles/_footer.scss:65]: Newline before block.

File: styles/_footer.scss [hunk 19]
```diff
@@ -1,20 +1,167 @@
...
+      }
+    }
+    address {
+      overflow: hidden;
+      display: block;
+      margin: 0;
+      font-style: normal;
+    }
+  }
+  .footer-nav {
```
Comment (ahmed-arbisoft) [inline styles/_footer.scss:59]: Newline before block.

File: styles/_footer.scss [hunk 20]
```diff
@@ -1,20 +1,167 @@
...
+      margin: 0 auto 15px;
+      @include tablet {
+        margin: 5px 15px 0 0;
+        float: left;
+      }
+      @include tablet {
+        margin: 5px 25px 0 0;
+      }
+    }
+    address {
```
Comment (ahmed-arbisoft) [inline styles/_footer.scss:51]: Newline before block.

File: styles/_footer.scss [hunk 21]
```diff
@@ -1,20 +1,167 @@
...
+      text-align: left;
+    }
+    img {
+      width: 78px;
+      margin: 0 auto 15px;
+      @include tablet {
+        margin: 5px 15px 0 0;
+        float: left;
+      }
+      @include tablet {
```
Comment (ahmed-arbisoft) [inline styles/_footer.scss:46]: Newline before block.

File: styles/_footer.scss [hunk 22]
```diff
@@ -1,20 +1,167 @@
...
+    @include tablet {
+      padding: 0;
+      float: left;
+      width: 285px;
+      text-align: left;
+    }
+    img {
+      width: 78px;
+      margin: 0 auto 15px;
+      @include tablet {
```
Comment (ahmed-arbisoft) [inline styles/_footer.scss:41]: Newline before block.

File: styles/_footer.scss [hunk 23]
```diff
@@ -1,20 +1,167 @@
...
+    line-height: 22px;
+    text-align: center;
+    padding: 0 0 20px;
+    @include tablet {
+      padding: 0;
+      float: left;
+      width: 285px;
+      text-align: left;
+    }
+    img {
```
Comment (ahmed-arbisoft) [inline styles/_footer.scss:37]: Newline before block.

File: styles/_footer.scss [hunk 24]
```diff
@@ -1,20 +1,167 @@
...
     display: inline-block;
     font-weight: $font-weight-light;
   }
+
+  .company-info {
+    font-size: 12px;
+    line-height: 22px;
+    text-align: center;
+    padding: 0 0 20px;
+    @include tablet {
```
Comment (ahmed-arbisoft) [inline styles/_footer.scss:30]: Newline before block.

File: styles/_footer.scss [hunk 25]
```diff
@@ -1,20 +1,167 @@
...
-  a { color: $secondary-text-color; }
+  a { color: white; }
 
   &-inner {
     @include max-width-container;
-    display: flex;
-    justify-content: space-between;
+    overflow: hidden;
+  }
+  .footer-inner {
```
Comment (ahmed-arbisoft) [inline styles/_footer.scss:13]: Newline before block.

File: styles/_header.scss [hunk 26]
```diff
@@ -6,12 +6,18 @@ $header-height: 71px;
...
   justify-content: space-between;
+  padding-top: 20px !important;
+  padding-bottom: 20px !important;
 }
 
-.logo img {
-  max-height: 37px;
+.logo {
+  width: 170px;
+  img {
```
Comment (ahmed-arbisoft) [inline styles/_header.scss:17]: Newline before block.

File: styles/_hero.scss [hunk 27]
```diff
@@ -1,16 +1,28 @@
...
-  background-image: url($homepage_background_image);
-  background-position: center;
-  background-size: cover;
-  height: 300px;
+  background: #2a2929 url($mit_background) repeat-x 50% 100%;
+  height: 170px;
   padding: 0 20px;
   text-align: center;
   width: 100%;
+  @include tablet {
```
Comment (ahmed-arbisoft) [inline styles/_hero.scss:10]: Newline before block.

**Review by ahmed-arbisoft — CHANGES_REQUESTED** [review]
Primarily SCSS formatting

File: manifest.json [hunk 28]
```diff
@@ -227,5 +239,62 @@
...
+      "type": "text",
+      "description": "mit_privacy_policy_description",
+      "label": "mit_privacy_policy_label",
+      "value": ""
+    }, {
+      "identifier": "mit_honor_code",
+      "type": "text",
+      "description": "mit_honor_code_description",
+      "label": "mit_honor_code_label",
+      "value": ""
```
Comment (asadiqbal08) [inline manifest.json]: @pdpinch any default ones for these empty values, please mention so I can update it here. 
currently, I leave them as an empty text field for the administrator.

File: manifest.json [hunk 29]
```diff
@@ -227,5 +239,62 @@
...
+      "type": "text",
+      "description": "mit_about_us_description",
+      "label": "mit_about_us_label",
+      "value": ""
+    }, {
+      "identifier": "mit_tos",
+      "type": "text",
+      "description": "mit_tos_description",
+      "label": "mit_tos_label",
+      "value": ""
```
Comment (pdpinch) [inline manifest.json]:
```suggestion
      "value": "https://xpro.mit.edu/terms-of-service/"
```

File: manifest.json (hunk 28 above)
Comment (pdpinch) [inline manifest.json]:
```suggestion
      "value": "https://xpro.mit.edu/honor-code/"
```

File: manifest.json [hunk 30]
```diff
@@ -227,5 +239,62 @@
...
+    }, {
+      "identifier": "mit_privacy_policy",
+      "type": "text",
+      "description": "mit_privacy_policy_description",
+      "label": "mit_privacy_policy_label",
+      "value": ""
```
Comment (pdpinch) [inline manifest.json]:
```suggestion
      "value": "https://xpro.mit.edu/privacy-policy/"
```

File: manifest.json [hunk 31]
```diff
@@ -227,5 +239,62 @@
...
+      "type": "text",
+      "description": "mit_phone_number_description",
+      "label": "mit_phone_number_label",
+      "value": "617-253-1000"
+    }, {
+      "identifier": "mit_home",
+      "type": "text",
+      "description": "mit_home_description",
+      "label": "mit_home_label",
+      "value": ""
```
Comment (pdpinch) [inline manifest.json]:
```suggestion
      "value": "https://www.mit.edu/"
```

**Review by pdpinch — CHANGES_REQUESTED** [review]
Are the missing defaults meant to be URLs? I gave you some suggestions.

**Comment by asadiqbal08** [comment]
One question left above beside, 
@ahmed-arbisoft @pdpinch are we good to merge  ?

**Review by pdpinch — APPROVED** [review]
I'm good with merging this. 

What happens after that? Can we load update ZenDesk to use this theme instead of the current one (and switch back if necessary)?

**Review by ahmed-arbisoft — APPROVED** [review]

**Pull request merged**